tests/robot-tests/tests/libs/slack.py

```diff
@@ -39,21 +47,34 @@ def _generate_slack_attachments(env: str):
                     "title": "Passed tests",
                     "value": passed_tests
                 },
-                {
-                    "title": "Failed tests",
-                    "value": failed_tests if failed_tests else "0"
-                },
+                failed_tests_field,
                 {
                     "title": "Results",
                     "value": "Failed" if failed_tests else "Passed"
                 },
-
             ]
         }
     ]
 
 
+def _get_report_status():
+    with open(f'{PATH}{os.sep}output.xml', 'rb') as report:
+        contents = report.read()
+
+        soup = BeautifulSoup(contents, 'lxml')
+        test = soup.find('total').find('stat')
+
+        failed_tests = int(test['fail'])
+
+        if failed_tests:
+            return True
+        else:
+            return False
+
+
 def send_slack_report(env: str):
+    should_send_report = _get_report_status()
+
     attachments = _generate_slack_attachments(env)
     data = {"attachments": attachments}
 
```

Comment:
I noticed a couple of typos on line 84 and 85 but github doesn't let me comment on those lines, so I'm commenting here: "needs to bet set" -> "needs to be set"